File: backend/src/main/java/ProgramaClip/ClipMoneyAPI/entidad/Cuenta.java
package ProgramaClip.ClipMoneyAPI.entidad;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

import com.sun.istack.NotNull;

import lombok.Data;

@Entity
@Data
public class Cuenta {
	@Id
	@GeneratedValue
	private Long nroCuenta;
	@Column(unique = true)
	@NotNull
	private String cvu;
	@NotNull
	private Float saldo;
	@ManyToOne
	@NotNull
	private Moneda moneda;
}

File: Models/Cuenta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace proyecto_clip_money_e_proyecto_clip.Models
{
    public class Cuenta
    {
        private long nroCuenta;
        private string cvu;
        private float saldo;
        private Moneda moneda;

        public Cuenta(long nroCuenta, string cvu, float saldo, Moneda moneda)
        {
            this.NroCuenta = nroCuenta;
            this.Cvu = cvu;
            this.Saldo = saldo;
            this.Moneda = moneda;
        }

        public long NroCuenta { get => nroCuenta; set => nroCuenta = value; }
        public string Cvu { get => cvu; set => cvu = value; }
        public float Saldo { get => saldo; set => saldo = value; }
        public Moneda Moneda { get => moneda; set => moneda = value; }
    }
}